payload.ts

```diff
@@ -10,7 +11,7 @@ type Transaction = {
   chainId?: string
 }
 
-export function create (method: string, params: readonly any[] = [], id: number, targetChain?: string): Payload {
+export function create (method: string, params: readonly unknown[] = [], id: number, targetChain?: string): Payload {
```

Comment:
should we allow creation of a payload without params, as in making it optional here?  Will need to change the signature if so, optional args need to go at the end

Reply:
this is an internal method so I don't think we'll ever need to create a payload with no params